hosted/src/NodeAllocator.cc

```diff
@@ -218,154 +221,122 @@ void ebbrt::NodeAllocator::DoAccept(
       }));
 }
 
+void ebbrt::NodeAllocator::AppendArgs(std::string arg) {
+  if (!cmdline_.empty()) {
+    cmdline_ += std::string(";");
```

Comment:
I'd be tempted to change it to a space since 1) Linux's command line works that way and 2) Space is the default separator used when parsing strings in C++